source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,71 @@
+
+[5]: https://github.com/alphagov/publishing-api/blob/master/lib/expansion_rules.rb#L29
+[6]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[7]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Tagging
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][8] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][9] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
+
+These rules are enforced by Whitehall but not by content-tagger meaning that these rules are not enforced uniformly.
+
+[8]: https://github.com/alphagov/whitehall/blob/56006c6f6ba033fbe450ef91d46204499e62e337/config/worldwide_tagging_organisations.yml
+[9]: https://github.com/alphagov/whitehall/blob/master/app/models/edition/taggable_organisations.rb#L4-L9
+
+## Concerns
+
+The separation of the world taxonomy has led it to become somewhat of a contentious issue. There have been a number of protracted discussions about whether this concept represents [technical debt for GOV.UK][10] tech community or if it is a form of product debt. Their existence is not well known among people who work on GOV.UK, despite it being a knowledge prerequisite to do any development work with the regular topic taxonomy.
+
+We also have a format of [world location][11] that was the predecessor to the world taxon pages. This is still used for a number of things ([atom feed][12], tagging in Whitehall, rendering the [world page][13]) so we’ve effectively got two concepts of the same thing and need to always keep them in sync.
+
+They are rendered in a completely different format to regular topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
+
+Content associated with a world location is tagged to the previous concept of a world location rather than a world taxon. For example https://www.gov.uk/government/news/information-and-events-for-uk-nationals-living-in-portugal has no data link to https://www.gov.uk/world/portugal and instead only has a link to the now defunct Portugal world location page. A [manual process][14] then creates a web link based on url guesswork.
+
+World taxons are treated the same as topic taxons in content tagger, the only clue a user has that they are dealing with something different is the /world part of their path. GOV.UK has 3921 published taxons, of those 2483 are world taxons meaning that they are the dominant theme in content-tagger.
```

Comment:
I'd probably put something about that 'at the time of writing (date) GOV.UK has x taxons' as this information is likely to get out of date over time.

Reply:
yeah I really should have checked this sooner rather than just blindly copying. I'm seeing 150/1002 which seems more realistic